server/routerlicious/packages/routerlicious-base/src/riddler/tenantManager.ts

```diff
@@ -287,6 +351,7 @@ export class TenantManager {
 		storage: ITenantStorage,
 		orderer: ITenantOrderer,
 		customData: ITenantCustomData,
+		enableKeylessAccess = false,
 	): Promise<ITenantConfig & { key: string }> {
 		const latestKeyVersion = this.secretManager.getLatestKeyVersion();
 
```

Comment:
The function above could be called "createTenantKeys" and it could be called twice. Once for non-private keys and once for private keys.   Lines 358 to 376 would not be required.